### PR TITLE
Team 4 add featured flag

### PR DESCRIPTION
**Pull request by HaonanJia**

### CODE REVIEW

File: srv/routes/reviews.js [hunk 1]
```diff
@@ -4,55 +4,73 @@ const HTTP_CREATED = 201
 const HTTP_CONFLICT = 409
 
 module.exports = function (app) {
-
+
```
Comment (kheyalimitra) [inline srv/routes/reviews.js:7]: extra space

File: scaling-rules.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (kheyalimitra) [inline scaling-rules.json]: This is from our change. right?

File: srv/routes/reviews.js [hunk 3]
```diff
@@ -4,55 +4,73 @@ const HTTP_CREATED = 201
...
+			res.send(result)
+		}
+		else
+		{
+			req.logger.info("Recovering Reviews")
+			res.send("App under maintenance")
+		}
 	})
-
+
```
Comment (kheyalimitra) [inline srv/routes/reviews.js:23]: same here,

File: srv/routes/reviews.js [hunk 4]
```diff
@@ -4,55 +4,73 @@ const HTTP_CREATED = 201
...
+	
 	app.get('/api/v1/reviews/:revieweeEmail', async function readAll(req, res) {
 		const revieweeEmail = req.params.revieweeEmail;
 		req.logger.info("Recovering all Reviews from %s", revieweeEmail)
 		const result = await app.reviewsService.getAllFor(revieweeEmail, req)
 		req.logger.debug("Result: %s", JSON.stringify(result))
 		res.send(result)
 	})
-
+
```
Comment (kheyalimitra) [inline srv/routes/reviews.js:31]: and here

File: srv/routes/reviews.js [hunk 5]
```diff
@@ -4,55 +4,73 @@ const HTTP_CREATED = 201
...
+			try {
+				await app.reviewsService.create(req.body, req)
+			} catch (err) {
+				req.logger.error("Review could not be posted: %s", err)
+				return res.status(HTTP_CONFLICT).end()
+			}
+			const reviewee_email = req.body.reviewee_email
+			await app.reviewsService.getAverageRating(reviewee_email)
+			req.logger.info("Review posted successfully")
+			res.status(HTTP_CREATED).location(req.body.component_name).end()
```
Comment (kheyalimitra) [inline srv/routes/reviews.js:54]: indentation issue?

**Review by kheyalimitra — APPROVED** [review]
Minor changes (related to space and indentation.) BTW, I see My changes added to this PR, Please avoid that.

**Comment by broxmen** [comment]
Closing this PR due to conflict, will be taken care in branch

**Pull request closed without merging**